internal/services/storage/storage_container_resource_test.go

```diff
@@ -35,7 +56,56 @@ func TestAccStorageContainer_basic(t *testing.T) {
 	})
 }
 
+func TestAccStorageContainer_complete(t *testing.T) {
```

Comment:
Should this also be renamed since it's being skipped?
```suggestion
func TestAccStorageContainer_completeDeprecated(t *testing.T) {
```

Reply:
Copy-paste error, this is a new test and should not have been skipped.